app/uk/gov/hmrc/tariffclassificationfrontend/views/advanced_search.scala.html

```diff
@@ -79,12 +78,15 @@ <h3 class="heading-medium">Filter search</h3>
      )
     </div>
 
-    <div class="column-three-quarters">
-     <h3 class="heading-medium" id="advanced_search-results_heading">Showing @{results.get.size} @if(results.get.size == 1){case}@if(results.get.size != 1){cases}</h3>
-     <hr class="mt-0"/>
-     @advanced_search_results(results.get)
-    </div>
-    </div>
+    @defining(results.get.filter(_.application.isBTI)) { searchResults: Seq[Case] =>
```

Comment:
I moved the filter up a level in case you're wondering. Liability orders are filtered out of the results, but were still getting shown in the "Showing x cases" count